movement/spring.py

```diff
@@ -205,9 +205,8 @@ def assemble_stiffness_matrix(self, boundary_conditions=None):
             # Determine boundary subsets for the associated tags
             bnd = self.mesh.exterior_facets
             tags = boundary_condition.sub_domain
-            if tags == "on_boundary":
+            if tags == ("on_boundary",):
                 tags = bnd.unique_markers
-            tags = [tags] if not isinstance(tags, Iterable) else tags
```

Comment:
Pretty sure `isinstance(tags, Iterable)` is now always `True`

Reply:
Yes I think you're correct.